app/mailers/application_mailer.rb

```diff
@@ -15,7 +15,7 @@ def mail(headers = {}, &block)
 
   class GroupMail
     def initialize(emails)
-      ActiveSupport::Deprecation.warn 'GroupMail will be removed as mailers should not generate multiple messages, use MailNotification#deliver'
+      ForemanDeprecation.deprecation_warning("MailNotification","deliver","1.10.0","GroupMail will be removed as mailers should not generate multiple messages")
```

Comment:
Same comment on all of these changes - code without spacing around arguments is hard to read, please include it as standard.